### PR TITLE
Adds the dmitool icon merger from baystation12

### PR DESCRIPTION
**Pull request by tkdrg**
Credits to GinjaNinja32@bay12 for developing dmitool and
PsiOmegaDelta@bay12 for some of these scripts. Thanks also to
neersighted for reminding me that we need this.

Basically this is like the map merger, except for icons. You can install
a hook using the instructions in the dmitool folder, and it should
automatically merge your DMIs for you. This means you can solve icon
conflicts effortlessly most of the time.

### CODE REVIEW

**Comment by duncathan** [comment]
:+1::x::100:

**Comment by PKPenguin321** [comment]
neat :+1:

File: tools/dmitool/merging.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+1. Install java(http://www.java.com/en/download/index.jsp)
+2. Make sure java is in your PATH. To test this, open git bash, and type "java". If it says unknown command, you need to add JAVA/bin to your PATH variable (A guide for this can be found at https://www.java.com/en/download/help/path.xml ).
+
+Merging
+The easiest way to do merging is to install the merge driver. For this, open `Baystation12/.git/config` in a text editor, and paste the following lines to the end of it:
```
Comment (N3X15) [inline tools/dmitool/merging.txt]: > Baystation12

Comment (tkdrg) [inline tools/dmitool/merging.txt]: > nitpicking

**Pull request merged**